src/models/MySqlModel.ts

```diff
@@ -57,10 +100,11 @@ export class MySqlModel {
 
     // The dataSource, logger and sqlStatement are required so bail if they are not provided
     if (dataSources && logger && dataSources.sqlDataSource && sqlStatement) {
-      const logMessage = `${reference}, sql: ${sqlStatement}, vals: ${values}`;
+      const sql = sqlStatement.split(/[\s\t\n]+/).join(' ');
```

Comment:
L103: Since this functionality is used multiple times, you might want to put it into a shared method.